asterisk_logcall/README.rst

```diff
@@ -0,0 +1,6 @@
+Asterisk Phone Log-call
```

Comment:
@treveradams   Thanks for your contribution to the OCA!
Please check the Travis output as there might errors/warnings to be corrected
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
* Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
* We recommend to include a screenshot as well to help new comers (nice to have)
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3